includes/general.php

```diff
@@ -2371,6 +2372,7 @@ function pods_no_conflict_on( $object_type = 'post', $object = null ) {
 				$no_conflict['filter'] = array_merge( $no_conflict['filter'], array(
 					array( 'add_term_metadata', array( PodsInit::$meta, 'add_term_meta' ), 10, 5 ),
 					array( 'update_term_metadata', array( PodsInit::$meta, 'update_term_meta' ), 10, 5 ),
+					array( 'update_term_metadata_by_id', array( PodsInit::$meta, 'update_term_meta_by_id' ), 10, 4 ),
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: f9596bf81f8eccf48222c6dc555a2a48</sub>